src/components/About/About.js

```diff
@@ -3,10 +3,16 @@ import Grid from '@material-ui/core/Grid';
 import {Typography} from '@material-ui/core';
 
 import './About.scss'
-
+import Bar from '../BottomBar/BottomBar'
 function About() {
 
+    const bar = {
+        title: 'ABOUT',
+        path: ''
+    }
     return (
+        <>
+        <Bar props={bar}></Bar>
```

Comment:
Props is general name for all properties that are propagated to React components. 
This should be `<Bar title={bar.title} path={bar.path} />` and Bar component will accept an object with keys `title` and `path`.